core/src/components/back-button/test/standalone/back-button.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('back-button: standalone', () => {
```

Comment:
Since `ion-back-button` doesn't reference `ion-app` directly, we can just get rid of the `standalone` test. ([Discussion for context](https://ionicteam.slack.com/archives/C02U5GFFXED/p1661529392273629)) You'll likely want to just delete the screenshots manually as well.